newrelic-fluent-bit-output.yaml

```diff
@@ -0,0 +1,38 @@
+package:
+  name: newrelic-fluent-bit-output
+  version: 1.16.0
+  epoch: 0
+  description: A Fluent Bit output plugin that sends logs to New Relic
+  copyright:
+    - license: Apache-2.0
+
+environment:
+  contents:
+    packages:
+      - wolfi-baselayout
+      - busybox
+      - build-base
+      - go
+      - ca-certificates-bundle
+
+pipeline:
+  # We can't use go/install because this requires specific ldflags to set the version
+  - uses: git-checkout
+    with:
+      repository: https://github.com/newrelic/newrelic-fluent-bit-output
+      tag: v${{package.version}}
+      expected-commit: 8e3aa7c9d829d2c36d8007c2d44295cefd5ee284
+
+  - runs: |
+      ARCH=$(go env GOARCH)
+      make VERSION=v${{ package.version }} linux/${ARCH}
+      mkdir -p ${{targets.destdir}}/newrelic/newrelic-fluent-bit-output
+      cp ./out_newrelic-linux-${ARCH}-v${{ package.version }}.so ${{targets.destdir}}/newrelic/newrelic-fluent-bit-output
```

Comment:
In the upstream image this seems to be [`/fluent-bit/bin/out_newrelic.so`](https://github.com/newrelic/newrelic-fluent-bit-output/blob/8e3aa7c9d829d2c36d8007c2d44295cefd5ee284/Dockerfile#LL24C83-L24C117), can we do there here too?

Reply:
I think the upstream filename is better, out_newrelic.so